### PR TITLE
Fix tests and add dot env

### PR DESCRIPTION
**Pull request by alanfriedman**
- Fixes broken tests by increasing mocha timeout, replacing PDF parser with `pdf-parse` and fixing some PDF text character encoding issues
- Adds [dotenv](https://github.com/motdotla/dotenv) for easily loading environment variables during local development

### CODE REVIEW

File: test/test-all.js [hunk 1]
```diff
@@ -17,21 +17,12 @@ BPromise.config({
...
-      reject(err);
-    });
-    pdfParser.on('pdfParser_dataReady', () => {
-      resolve(pdfParser.getRawTextContent());
-    });
-
-    pdfParser.parseBuffer(buffer);
-  });
+  // Fix for characters encoding issues
+  return pdf(buffer).then(data => data.text.replace(/[\W_]+/g, '-'));
```
Comment (alanfriedman) [inline test/test-all.js]: Replace all non-alphanumeric characters with a hyphen to resolve some difference in character encoding when comparing strings extracted from the PDF and strings defined in the test environment

**Comment by alanfriedman** [comment]
Partial dupe of #103. If we merge that one I can delete everything except for dotenv from this PR.

**Comment by kimmobrunfeldt** [comment]
Thanks for the effort! The other fixes are very welcome, but don't want to add dotenv changes at this point. The other changes were addressed mostly in #103 except for the timeout increases and the character replacements. Those would be helpful and I would happily merge them to the project.


dotenv: it's definitely a great tool, but I'd prefer not to change the env mechanism now. At some point I might add the dotenv with a few other features like env type checking, which I've used in other projects.

**Comment by kimmobrunfeldt** [comment]
@alanfriedman I copy-pasted the character replacement part from this PR to master and fixed it while doing other changes. I would've rather took your commit for it to give the correct credit but my time is quite short so I need to do all fixes now, otherwise I'll come back to this PR in a few months and it will delay. 

Thanks for the PR in any case, it was helpful!

**Pull request closed without merging**